importer/main.py

```diff
@@ -974,5 +1052,6 @@ def main(
     total_time = end_time - start_time
     logging.info("Total time: " + str(total_time.total_seconds()) + " seconds")
 
+
```

Comment:
remove

Reply:
When I black format the files the spaces are added back